jwst/resample/resample_utils.py

```diff
@@ -135,7 +134,7 @@ def calc_gwcs_pixmap(in_wcs, out_wcs):
     """ Return a pixel grid map from input frame to output frame.
     """
     # TODO: Is the following 1-indexed or 0-indexed?  Check.
-    grid = wcstools.grid_from_domain(in_wcs.domain)
+    grid = wcstools.grid_from_bounding_box(in_wcs.bounding_box, step=(1, 1))
```

Comment:
Is it also necessary to use `center=True` to get the grid at the center of the pixels?